examples/svelte/kitchen-sink/src/index.css

```
@@ -0,0 +1,328 @@
+}
+
+.demo-button,
+.pin-button,
+.compact-input,
+.filter-input,
+.filter-select,
+.page-size-input,
+.text-input,
+.number-input,
+.icon-button {
+  border: 1px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)